HW 6/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.9"
+
+services:
+  db:
+    image: mariadb
+    command: mysqld --default-authentication-plugin=mysql_native_password
+    restart: always
+    volumes:
+      - mysql_data:/var/lib/mysql
```

Comment:
I would use "mysql_data:/config" inside the container because the volume is managed by docker and therefore has better performance. '/config' directory inside MariaDB contains *.cnf, and all databases (as subdirectories).